src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -279,8 +279,7 @@ void CascadiaSettings::_CreateDefaultKeybindings()
                               KeyChord{ KeyModifiers::Ctrl | KeyModifiers::Shift,
                                         static_cast<int>('9') });
     keyBindings.SetKeyBinding(ShortcutAction::NewTabProfile9,
-                              KeyChord{ KeyModifiers::Ctrl | KeyModifiers::Shift,
-                                        static_cast<int>('0') });
+                              KeyChord{ /* unbound, because CTRL + SHIFT + 0 is a system shortcut */ });

```

Comment:
I'd maybe just remove the entire `SetKeyBinding` call here, I'm not sure what binding an action to vkey=0 would do.

Reply:
(but leave the comment that ShortcutAction::NewTabProfile9 is unbound, because CTRL + SHIFT + 0 is a system shortcut